build/Targets/VSL.Versions.targets

```diff
@@ -61,14 +61,13 @@
 
   <!-- NuGet version -->
   <PropertyGroup>
-    <VSSemanticVersion>15.0.1</VSSemanticVersion>
+    <VSSemanticVersion>15.0</VSSemanticVersion>
```

Comment:
Sorry, I meant shouldn't we keep the NuGet version same as VSIX and assembly versions?